src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/wp-cli.yml')) {
+            return 'WordPress';
+        }
+        else {
+            // This upstream works for Drupal 8 and Drupal 7.
```

Comment:
Actually, 'Empty Upstream' only works for Drupal 8. I made another upstream named 'Empty 7' for Drupal 7. Maybe test for "$siteDir/web/misc/drupal.js" as an indicator that the site is Drupal 7.